src/courier.js

```diff
@@ -44,17 +47,22 @@ const stopAndLog = (log) => {
   console.log(timedLog)
 }
 
-const logToConsole = (level, origin, message) => {
+const logToConsole = (level, origin, message, timeout, action) => {
   const isLogLevelInfo = log.level === 'info'
-  const {message: text, timeout} = typeof message === 'string'
+  const {message: text, timeout: msgTimeout} = typeof message === 'string'
     ? {message} : message
   if (isLogLevelInfo && isSpinnerActive()) {
     if (level === 'error') {
       stopAndLog(consumeChangeLog())
       clearAbove()
       return console.log(`\n${text}\n`)
     }
-    setSpinnerText(text, timeout)
+
+    if (timeout) {
```

Comment:
That's ok, msgTimeout it's not meant to trigger the courier timeout.
As you see below, it's was created for controlling the spinner on `watch`